frontend/src/components/TopicCards/TopicCards.jsx

```
@@ -1,30 +1,29 @@
-              key={topic.topic_id}
-              topic={topic}
-              onClick={() => handleTopic(topic.category, topic)}
-            />
-          </div>
-        ))}
+      <div className="font-medium mb-5 text-center">주제를 선택해주세요</div>
+      <div className="flex justify-center">
+        <div className="grid grid-cols-2 gap-4">
+          {topics.map(topic => (
+            <div className="bg-cyan-500/70 rounded-lg shadow-lg p-3 w-[150px] h-[150px] flex items-center justify-center">
+              <TopicCard
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)